### PR TITLE
fix: redirect URL preventing document flow

### PR DESCRIPTION
**Pull request by dguyen**
## Description

Currently the document redirect URL feature is preventing documents from being created unless a redirect URL is provided.

During the document edit flow, the redirect URL is hidden in an advanced tab with the value of an empty string, which will always fail the current Zod validation since `optional` requires undefined to pass.

There are multiple ways to fix this, but I think this is the easiest method where we can assume an empty string is valid.

## Testing Performed

Will require retesting the feature since I'm just assuming an empty string won't break anything downstream for the redirect URL feature.

Pinging @adithyaakrishna to retest the feature.

### CODE REVIEW

File: packages/trpc/server/document-router/schema.ts [hunk 1]
```diff
@@ -74,12 +73,7 @@ export const ZSendDocumentMutationSchema = z.object({
     message: z.string(),
     timezone: z.string(),
     dateFormat: z.string(),
-    redirectUrl: z
```
Comment (Mythie) [inline packages/trpc/server/document-router/schema.ts:77]: zod url's have issues apparently, it's probably easier to just update the refine to allow an empty string.

Comment (dguyen) [reply packages/trpc/server/document-router/schema.ts:77]: Any references?

Comment (Mythie) [reply packages/trpc/server/document-router/schema.ts:77]: On the OG PR

Comment (dguyen) [reply packages/trpc/server/document-router/schema.ts:77]: Got it 👍

**Review by Mythie — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
@dguyen Confirmed everything works 🚀

**Pull request merged**